app/models/issue.rb

```diff
@@ -24,16 +24,10 @@ def get_headers(header = nil)
     all_headers.find { |h| h[header] }.try(:values).try(:first)
   end
 
-  def stacktrace_frames
-    #added a switch statement in case errors from different platforms will be saved differenty
-    case platform
-    when 'javascript'
-      frames = get_interfaces(:stacktrace)._data[:frames]
-      frames = [] if frames.blank?
-    else
-      frames = get_platform_frames
-    end
-    frames
+  def breadcrumbs_stacktrace
+    breadcrumbs = get_interfaces(:breadcrumbs)
+    return 'Missing breadcrumbs' if breadcrumbs.blank?
```

Comment:
Stop returning messages from these methods. Return a boolean value, True, false or even nil is good.

You can see you had to verify if the stack trace is present in the view and you checked against the message that's really really bad!!.